src/Select/web/OptionList.js

```diff
@@ -8,7 +8,7 @@ const OptionList = List.extend`
   z-index: 1050;
   background: ${(props) => props.theme.color.white};
   min-width: ${(props) => props.block ? '100%' : props.theme.px(25)};
-  max-height: ${(props) => props.theme.px(27)};
+  max-height: ${(props) => props.theme.px(props.height)};
```

Comment:
Remove the min-width and max-width styles from here, it's no longer needed since it's dictated by the VirtualList itself